.github/workflows/build-wheel-linux-arm64.yaml

```
@@ -262,66 +293,158 @@ jobs:
+        path: ${{ github.workspace }}/standalone_plugin_wheel/dist
+        retention-days: 14
+
   test-wheels:
     needs: [constants, catalyst-linux-wheels-arm64]
     strategy:
       fail-fast: false
-      max-parallel: 1
       matrix:
-        python_version: [{major_minor: "3.10", patch: "14", package: "python3.10"},
-                         {major_minor: "3.11", patch: "9",  package: "python3.11"},
-                         {major_minor: "3.12", patch: "3",  package: "python3.12"}]
-        container_img: ["quay.io/pypa/manylinux_2_28_aarch64"]
-        container_name: ["manylinux_2_28_aarch64"]
+        python_version: ${{ fromJSON(needs.constants.outputs.python_versions) }}
 
     # To check all wheels for supported python3 versions
-    name: Test Wheels (Python ${{ matrix.python_version.major_minor }})
-    runs-on:
-      group: 'Office 24th floor M2 Mac'
+    name: Test Wheels (Python ${{ matrix.python_version }}) on Linux Arm
+    runs-on: ubuntu-22.04-arm
 
     steps:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v4
 
-    - name: Setup Runner Environment
-      id: setup_env
-      uses: ./.github/workflows/utils/setup_self_hosted_macos_env_linux_arm64
-      with:
-        python_version: ${{ matrix.python_version.major_minor }}
-
     - name: Download Wheel Artifact
       uses: actions/download-artifact@v4
       with:
-        name: catalyst-linux_arm64-wheel-py-${{ matrix.python_version.major_minor }}.zip
+        name: catalyst-linux_arm_64-wheel-py-${{ matrix.python_version }}.zip
         path: ${{ github.workspace }}/dist
 
+    - name: Download Standalone Plugin Wheel Artifact
+      # Run only on Thursday at the given time
+      if: github.event_name == 'schedule'
+      uses: actions/download-artifact@v4
+      with:
+        name: standalone-plugin-linux_arm_64-wheel-py-${{ matrix.python_version }}.zip
+        path: ${{ github.workspace }}/standalone_plugin_wheel/wheel
+
+    - name: Set up Python ${{ matrix.python_version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python_version }}
+
+    - name: Install Python dependencies
+      run: |
+        python${{ matrix.python_version }} -m pip install pytest pytest-xdist pytest-mock
+
+    - name: Install PennyLane Plugins
+      run: |
+        python${{ matrix.python_version }} -m pip install PennyLane-Lightning-Kokkos
+        python${{ matrix.python_version }} -m pip install 'amazon-braket-pennylane-plugin>=1.31.0'
+
+    - name: Install OQC client
+      run: |
+        python${{ matrix.python_version }} -m pip install oqc-qcaas-client
+
+    - name: Install Catalyst
+      run: |
+        python${{ matrix.python_version }} -m pip install dist/*.whl --extra-index-url https://test.pypi.org/simple
+
+    - name: Install Standalone Plugin
+      # Run only on Thursday at the given time (TODO: set comparison to == before merging)
+      if: github.event_name == 'schedule'
```

I'm guessing this todo is resolved now? 
```suggestion
      # Run only on Thursday at the given time
      if: github.event_name == 'schedule'
```